### DISCUSSION TITLE
Compiling using install.md

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#4420] [Q&A]
I am trying to compile an Android version and am following the very good instructions in install.md.
Managed to install Android Studio ok but stuck at how to configure the repo etc.
<img width="661" alt="Screenshot 2023-02-09 at 2 46 20 pm" src="https://user-images.githubusercontent.com/58628197/217846907-38648fc3-b797-4527-8f88-f1431ffde608.png">
I am using a Macbook and when I try to run the macOS statement as above I get
% ./tools/android/set_up_android.py --sdk $HOME/Library/Android/sdk
zsh: no such file or directory: ./tools/android/set_up_android.py
Any help grateful

**Comment by biodranik** [comment]
~You don't need to run this script. Try without it, and delete it's mentioning from the documentation if it works.~

I was wrong, the script is still required.

**Reply by rtsisyk** [reply]
> You don't need to run this script. Try without it, and delete it's mentioning from the documentation if it works.

It doesn't work on MacOS, I tried a couple of weeks ago.
